config.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"io/ioutil"
+
+	"github.com/go-yaml/yaml"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/go-yaml/yaml (can't find import: github.com/go-yaml/yaml)
* [aligncheck] "github.com/go-yaml/yaml" imported but not used as error
* [structcheck] unused struct field "github.com/go-yaml/yaml" imported but not used as error
* [varcheck] unused global variable "github.com/go-yaml/yaml" imported but not used as error